src/librustc_mir/interpret/intrinsics/caller_location.rs

```diff
@@ -54,9 +54,14 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
     }
 
     pub fn alloc_caller_location_for_span(&mut self, span: Span) -> MPlaceTy<'tcx, M::PointerTag> {
+        let (file, line, column) = self.location_triple_for_span(span);
+        self.alloc_caller_location(file, line, column)
+    }
+
+    pub fn location_triple_for_span(&self, span: Span) -> (Symbol, u32, u32) {
```

Comment:
If this is not used anywhere else (like it seems), please make it private.

Reply:
It's used elsewhere in the crate: https://github.com/rust-lang/rust/pull/67137/files#diff-1a04d235d1077ab50fdc4e8ce54d230eR383

Reply:
Ah, not sure why I didn't see it then.
That's still inside the interpreter though, so please make it `pub(super)`.